notebooks/en/enterprise_cookbook_gradio.ipynb

```diff
@@ -0,0 +1,5796 @@
+{
```

Comment:
<p>You can probably just shorten the section title to "A meeting transcription tool" and then all the subsections can be: Audio-to-text, Organize &amp; summarize text, Putting it all together. No need to add Transcript tool at the beginning since it's already clear what the reader is making</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>